desertbot/modules/commands/Style.py

```diff
@@ -79,6 +79,33 @@ def onLoad(self):
                 ord('y'): '𝖞',    ord('Y'): '𝖄',
                 ord('z'): '𝖟',    ord('Z'): '𝖅',
             },
+            'smallcaps': {
+                ord('a'): 'ᴀ',
+                ord('b'): 'ʙ',
+                ord('c'): 'ᴄ',
+                ord('d'): 'ᴅ',
+                ord('e'): 'ᴇ',
+                ord('f'): 'ꜰ',
+                ord('g'): 'ɢ',
+                ord('h'): 'ʜ',
+                ord('i'): 'ɪ',
+                ord('j'): 'ᴊ',
+                ord('k'): 'ᴋ',
+                ord('l'): 'ʟ',
+                ord('m'): 'ᴍ',
+                ord('n'): 'ɴ',
+                ord('o'): 'ᴏ',
+                ord('p'): 'ᴘ',
+                ord('q'): 'ꞯ',
```

Comment:
I can't see this character no matter what font or editor I try. That could possibly become a problem. Looks good to me otherwise.

Reply:
From what i can tell there is no small capital q (https://www.fileformat.info/info/unicode/char/a7af/index.htm) in any common font, but there is ǫ (borrowed from another site), which should be close enough?

Reply:
I can see that one fine so we should probably use that insteaqd.